libraries/chain/block_header_state_legacy.cpp

```diff
@@ -107,37 +92,39 @@ namespace eosio { namespace chain {
       int32_t i = (int32_t)(result.confirm_count.size() - 1);
       uint32_t blocks_to_confirm = num_prev_blocks_to_confirm + 1; /// confirm the head block too
       while( i >= 0 && blocks_to_confirm ) {
-         --result.confirm_count[i];
-         //idump((confirm_count[i]));
-         if( result.confirm_count[i] == 0 )
-         {
+        --result.confirm_count[i];
```

Comment:
Why is the space removed?

Reply:
Just copied over the 5.0 impl, including spaces